data/lua/core/interface.lua

```diff
@@ -53,6 +53,11 @@ if wesnoth.kernel_type() == "Game Lua Kernel" then
 	wesnoth.zoom = wesnoth.deprecate_api('wesnoth.zoom', 'wesnoth.interface.zoom', 1, nil, wesnoth.interface.zoom)
 	wesnoth.gamestate_inspector = wesnoth.deprecate_api('wesnoth.gamestate_inspector', 'gui.show_inspector', 1, nil, gui.show_inspector)
 	wesnoth.color_adjust = wesnoth.deprecate_api('wesnoth.color_adjust', 'wesnoth.interface.color_adjust', 1, nil, function(cfg) wesnoth.interface.color_adjust(cfg.red, cfg.green, cfg.blue) end)
+	wesnoth.allow_end_turn = wesnoth.deprecate_api('wesnoth.allow_end_turn', 'wesnoth.interface.allow_end_turn', 1, nil, wesnoth.interface.allow_end_turn)
+	wesnoth.clear_messages = wesnoth.deprecate_api('wesnoth.clear_messages', 'wesnoth.interface.clear_messages', 1, nil, wesnoth.interface.clear_messages)
+	wesnoth.end_turn = wesnoth.deprecate_api('wesnoth.end_turn', 'wesnoth.interface.end_turn', 1, nil, wesnoth.interface.end_turn)
+	wesnoth.get_viewing_side = wesnoth.deprecate_api('wesnoth.get_viewing_side', 'wesnoth.interface.get_viewing_side', 1, nil, wesnoth.interface.get_viewing_side)
+	wesnoth.message = wesnoth.deprecate_api('wesnoth.message', 'wesnoth.interface.add_message', 1, nil, wesnoth.interface.add_message)
```

Comment:
Maybe make this `add_chat_message` to clarify further that it's unrelated to `[message]`?

Reply:
I'm not opposed to that, but it seems that if this one is called `add_chat_message` then the one that clears it should be `clear_chat_messages`, which means that `clear_chat_messages` also clears WML error messages. Is that okay? (Of course, it only clears them from chat; they'd still be in the Lua console.)

Reply:
I agree with the logic, and think the name `clear_chat_messages` while a little inaccurate will be clear to devs about which part of the UI it clears.

I do wonder if there should be a Lua function for clearing the chat area at all - it sounds like something that should only be triggered by the player choosing to do it.

Reply:
You might have a point? It's not used in mainline anywhere, either…